NANOProducer/plugins/NANOProducer.cc

```diff
@@ -212,7 +212,7 @@ NANOProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
     std::vector<float> mu_dzError;
     std::vector<float> mu_numberOfValidPixelHits;
     std::vector<float> mu_numberOfpixelLayersWithMeasurement;
-    //std::vector<float> mu_numberOfstripLayersWithMeasurement ;
+    std::vector<float> mu_numberOfstripLayersWithMeasurement ;
```

Comment:
are these floats or ints?

Reply:
it is int but i have tested it with float and there was no problem.   I thought that we need float variables type for the unpacker.